### PR TITLE
PR for Modal Component

### PR DESCRIPTION
**Pull request by manojasarsa**
PR for Modal Component

### CODE REVIEW

File: components/modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
+@import url("/docs/main.css");
+
+.modal_box {
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    right: 0;
+    left: 0;
```
Comment (vaibhav18matere) [inline components/modal/modal.css:8]: can make separate class for this

File: components/modal/modal.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    background-color: var(--white-color);
+}
+
+.modal {
+    width: 35vw;
+    height: 20vw;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
```
Comment (vaibhav18matere) [inline components/modal/modal.css]: can make separate class for this

File: components/modal/modal.html [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+                </iframe>
+            </div>
+
+            <h2 class="doc_main_head">Example of Modal</h2>
+
+            <button class="btn btn_secondary modal_btn_live" id="modal_btn_live">Live Demo</button>
+
+            <div class="modal_box" id="modal_box">
+                <div class="modal">
+                    <i class="fas fa-exclamation-triangle site-icon text_lg"></i>
```
Comment (vaibhav18matere) [inline components/modal/modal.html]: you can increase size of this triangle

**Review by vaibhav18matere — APPROVED** [review]

File: components/modal/modal.css [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    row-gap: 1rem;
+    border: 1px solid var(--primary-color);
+    background-color: var(--line-color);
+}
+
+.modal_active {
+    z-index: 1;
+    display: flex;
+    background-color: var(--);
```
Comment (ajitpsakri) [inline components/modal/modal.css]: Add a variable name to `background-color`

**Review by ajitpsakri — APPROVED** [review]

File: components/modal/modal.css (hunk 4 above)
Comment (manojasarsa) [reply components/modal/modal.css]: Done!

**Pull request merged**